examples/wireshark.py

```diff
@@ -9,7 +9,7 @@
 parses it and shows a short summary for every protocol.
 """
 from __future__ import print_function
-from pywinauto.application import Application
+from pywinauto.windows.application import Application
```

Comment:
It's better to leave as `from pywinauto.application import Application` for users. And it should work for both platforms, maybe except different executable names. But `Application().start("app.exe")` could be adjusted to Linux `Application().start("app")` automatically.